src/accessibility/gridOutput.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * @module Environment
+ * @submodule Environment
+ * @for p5
+ * @requires core
+ */
+import p5 from '../core/main';
+
+let ingredients = {};
+let background;
+
+//creates html structure
+p5.prototype._createGridOutput = function(cIdT) {
+  let inner =
+    'Grid Output<p id="' +
+    cIdT +
+    'Summary" aria-label="grid output summary"><table id="' +
+    cIdT +
+    'OD" summary="grid output content"></table><ul id="' +
+    cIdT +
+    'SD" aria-label="grid output shape details"></ul>';
+  return inner;
+};
+
+//creates output content
+p5.prototype._updateGridOutput = function(cnvId, ing, bkgr) {
+  ingredients = ing;
+  background = bkgr;
+  let cIdT = cnvId + 'grOut';
+  let innerSDs = this._buildGridSDs(cIdT);
+  let innerSummary = this._buildGrSummary(innerSDs.numShapes);
+  let innerMap = this._buildGrMap(cIdT);
+  if (innerSummary !== document.getElementById(cIdT + 'Summary').innerHTML) {
+    document.getElementById(cIdT + 'Summary').innerHTML = innerSummary;
+  }
+  if (innerMap !== document.getElementById(cIdT + 'OD').innerHTML) {
+    document.getElementById(cIdT + 'OD').innerHTML = innerMap;
+  }
+  if (innerSDs !== document.getElementById(cIdT + 'SD').innerHTML) {
+    document.getElementById(cIdT + 'SD').innerHTML = innerSDs.details;
+  }
+};
+
+//creates spatial grid
+p5.prototype._buildGrMap = function(cId) {
```

Comment:
to make this more readable and align with the naming of other functions (ie `_updateGridOutput`), I would suggest writing out `Grid`